tests/test_analogue.py

```diff
@@ -17,6 +17,7 @@
     ("xk_216_mc", "2AMi18o18mssaax"),
     ("xk_216_mc", "2ASi10o10xssxxx"),
     ("xk_evk_xu316", "1AMi2o2xxxxxx"),
+    ("xk_evk_xu316", "2AMi2o2xxxxxx_lp"),
```

Comment:
Would it be okay to add this to the dac loopback test smoke configs instead? Adding here would add 2 tests (input and output) to the smoke run while adding to test_loopback will add one. We want to keep the smoke runs as short as possible.

Reply:
I'm not sure we want to test LP on the EVK at all to be honest. I shall remove (we can't test dac loopback on EVK right?)